docs/workflow_solutions/getting_containers.md

```diff
@@ -287,6 +287,66 @@ python python_test.py
 
 More lessons on Docker can be found in this link: [Introduction to Docker](https://christinalk.github.io/docker-introduction/) and [Docker Documentation](https://docs.docker.com/reference/dockerfile/).
 
+## How to Access and Use Community Containers in the GitLab Container Registry
+
+In the UAB Research Computing [GitLab instance](../account_management/gitlab_account.md#uab-gitlab-registration), we provide certain community containers—prebuilt, these containers are standardized environments that grant you access to a variety of software tools. in the sections below, we share information for what you will find in the gitlab community container repo, and how you can access them.
+
+### Software We Support in Building Community Containers
+
+We provide community containers that cover a broad range of software to support your research needs. In general, the containers we have in our container registry are for software and applications with;
```

Comment:
"with the following."

I think generally ending the sentence is best if your list items are long.

Alternatively, use "with:" (colon, not semicolon), then lowercase the start of each list item, and append a semicolon `;` instead of a full stop at the end of each item. That will make it all flow like one sentence. However, because these list items are relatively long, it is probably best to stick to the first option, making each an independent clause or sentence.